### PR TITLE
Global ChilledCeilingPanelSimple

### PR DESCRIPTION
**Pull request by mitchute**
### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by mitchute** [comment]
Ready for review.

File: src/EnergyPlus/ChilledCeilingPanelSimple.hh [hunk 1]
```diff
@@ -52,54 +52,36 @@
...
+        ZoneConvectiveLoad  // Controls system using zone convective remaining load
+    };
 
-    // Types
+    // Condensation control types:
+    enum class CondCtrl {
+        NONE,               // Condensation control--none, so system never shuts down
+        SIMPLEOFF,          // Condensation control--simple off, system shuts off when condensation predicted
+        VARIEDOFF           // Condensation control--variable off, system modulates to keep running if possible
+    };
```
Comment (Myoldmopar) [inline src/EnergyPlus/ChilledCeilingPanelSimple.hh:84]: enum classes, awesome!

File: src/EnergyPlus/BranchNodeConnections.hh [hunk 2]
```diff
@@ -64,38 +64,24 @@ namespace DataBranchNodeConnections {
...
-
-    // DERIVED TYPE DEFINITIONS:
-    // na
-
-    // MODULE VARIABLE DECLARATIONS:
-    // na
-
-    // SUBROUTINE SPECIFICATIONS FOR MODULE
-
-    // Functions
```
Comment (Myoldmopar) [inline src/EnergyPlus/BranchNodeConnections.hh:79]: So many comments cleaned out.

File: src/EnergyPlus/HeatBalanceSurfaceManager.cc [hunk 3]
```diff
@@ -4388,7 +4388,7 @@ namespace HeatBalanceSurfaceManager {
     // Beginning of Record Keeping subroutines for the HB Module
     // *****************************************************************************
 
-    void UpdateFinalSurfaceHeatBalance()
+    void UpdateFinalSurfaceHeatBalance(ChilledCeilingPanelSimpleData &dataChilledCeilingPanelSimple)
```
Comment (Myoldmopar) [inline src/EnergyPlus/HeatBalanceSurfaceManager.cc:4391]: I know long term we won't be able to keep this as just the chilled ceiling, and that's fine. I like that we are keeping it as low-level as we can.

**Review by Myoldmopar — APPROVED** [review]
This looks great, enum classes are a big improvement, and a net reduction of over 400 lines of code, nice!

**Comment by Myoldmopar** [comment]
CI is super happy with this.  I pulled develop in the branch locally and rebuilt, everything builds and runs fine.  Good to go, merging!

**Pull request merged**